tinygrad/helpers.py

```diff
@@ -36,6 +38,17 @@ def partition(lst:List[T], fxn:Callable[[T],bool]):
   b:List[T] = []
   for s in lst: (a if fxn(s) else b).append(s)
   return a,b
+def temp(x:str) -> str: return (Path(tempfile.gettempdir()) / x).as_posix()
+def get_child(parent, key):
```

Comment:
Can this be a separate PR?

Reply:
Will do this in a separate PR. I'll link the PR number to this comment.